static/js/apps/explore_landing/topics.json

```diff
@@ -561,76 +612,72 @@
       "examples": {
         "general": [
           {
-            "title": "Which countries emit the most greenhouse gases",
-            "url": "oq=Which+countries+emit+the+most+greenhouse+gases"
+            "title": "Which counties emit the most greenhouse gasses",
+            "url": "oq=Which+counties+emit+the+most+greenhouse+gasses&p=country/USA&pt=County&t=dc/topic/GreenhouseGasses"
           },
           {
             "title": "Emissions in Harris County, Texas",
-            "url": "oq=Emissions+in+Harris+County,+Texas"
+            "url": "oq=Emissions+in+Harris+County,+Texas&p=geoId/48201&t=dc/topic/Emissions"
           },
           {
             "title": "Pollution in Los Angeles County",
-            "url": "oq=Pollution+in+Los+Angeles+County"
+            "url": "oq=Pollution+in+Los+Angeles+County&t=dc/topic/Pollution&p=geoId/06037"
           },
           {
-            "title": "Projected temperature extremes in India",
-            "url": "oq=Projected+temperature+extremes+in+India"
+            "title": "Projected temperature extremes in US counties",
+            "url": "oq=Projected+temperature+extremes+in+USA&p=country/USA&pt=County&t=dc/topic/TemperaturePredictions"
```

Comment:
Leave out the sustainability topics, they'll change?

Reply:
Removed. Switched each of these to start with `q` instead of `oq` since "q" looks like it runs the query against NL. Let me know if I should stick with `oq`instead.